Bottom: b6d5ba596f0c3fc0ca1aaffa410bc217d28a0240
Top:    b6d5ba596f0c3fc0ca1aaffa410bc217d28a0240
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 22:56:59 +0000

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***







---


